src/test/java/com/automationPractice/pages/MainPage.java

```diff
@@ -29,5 +33,29 @@ public String getTittleOfMainPage(){
     public void clickOnContactButton(){
         clickOnElement(BTN_ContactUs);
     }
+    public void sendSearch(String message){
```

Comment:
- Insert a empty line between lines 35 and 36. 
- There is missing the Java doc for this method.

Reply:
problem solved